services/src/main/java/uk/gov/moj/sdt/services/SubmitQueryService.java

```diff
@@ -133,7 +144,7 @@ public SubmitQueryService(@Qualifier("ConsumerGateway")
     /**
      * threshold for incoming requests for each target application.
      */
-    private Map<String, Integer> concurrentRequestsInProgress = new HashMap<String, Integer>();
+    private Map<String, Integer> concurrentRequestsInProgress = new HashMap<>();
```

Comment:
move this variable to before constructor

Reply:
Done